app/src/main/java/se/inera/intyg/minaintyg/logging/MonitoringLogService.java

```diff
@@ -9,4 +9,6 @@ public interface MonitoringLogService {
   void logListCertificates(String personId, int nbrOfCertificates);
 
   void logCertificateRead(String personId, String certificateId);
```

Comment:
Noticed that logCertificateRead has incorrect names of arguments... Could you change the argument names? Should be certificateId and type.